### PR TITLE
Attributes names that correspond with katcp weather sensors

### PR DESCRIPTION
**Pull request by Athanaseus**

### CODE REVIEW

File: tango_simlib/tests/weather_supplementary_SIMDD.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     },
     {
       "basicAttributeData": {
-        "name": "relative_humidity",
+        "name": "relative-humidity",
         "dataSimulationParameters": {
```
Comment (lvdheever) [inline tango_simlib/tests/weather_supplementary_SIMDD.json]: Try to get a naming convention for these interface description files. I suggest Weather.xmi and Weather_SIMDD.json. This aligns with Dish.xmi and Dish_SIMDD.json used for DishElement.
Or is there anything in weather_sim.xmi that is not the standard weather TANGO interface and is simulator specific? If so, then may be Weather_sim.xmi and Weather_SIMDD.json.

Comment (kmadisa) [inline tango_simlib/tests/weather_supplementary_SIMDD.json]: @lvdheever There is nothing special about this particular file (and its name). For testing purposes, we sort of created numerous files for the same simulator each time we added a new functionality to the tango-simlib framework.

Comment (Athanaseus) [inline tango_simlib/tests/weather_supplementary_SIMDD.json]: https://github.com/ska-sa/tango-simlib/pull/38

**Comment by Athanaseus** [comment]
Attributes naming that correspond with katcp weather sensors will be handle dynamically in the translator.

**Pull request closed without merging**